client/checkout/stripe-link/index.js

```diff
@@ -51,6 +51,17 @@ const enableStripeLinkPaymentMethod = ( options ) => {
 			  };
 
 		if ( options.complete_shipping() ) {
+			if (
```

Comment:
Can we omit this if, assuming the name is accessible - if Stripe already supplied it, it's unlikely they'll stop providing it without giving us a heads up.